api/src/main/resources/marquez/db/migration/V49__column_lineage.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE column_lineage (
   output_dataset_field_uuid     uuid REFERENCES dataset_fields(uuid),
   input_dataset_version_uuid    uuid REFERENCES dataset_versions(uuid), -- speed up graph column lineage graph traversal
   input_dataset_field_uuid      uuid REFERENCES dataset_fields(uuid),
-  transformation_description    VARCHAR(255),
+  transformation_description    TEXT,
```

Comment:
We can modify the column in existing migration file as long as it is not released.